docs/DeveloperGuide.adoc

```diff
@@ -723,25 +723,99 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new person | save my contacts in the App
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* *` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* *` |user |change the UI theme |customize my App
+|`*` |loyal user |send feedback to developers |help further improve this App
+|`*` |user |save my contacts on the cloud |sync between different devices
```

Comment:
This user story and the one above seems to necessitate an NFR that touches on availability of an internet connection?

Reply:
i think _sync and upload to the cloud_ should be a functionality?

Though part of it becomes a NFR.

Reply:
Yeah, I think what you have here is fine, just need an additional NFR to complement it.

Reply:
sure NFR added already.